src/imports/Ubuntu/AddressBook/ContactView/ContactDetailBirthdayView.qml

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (C) 2012-2015 Canonical, Ltd.
```

Comment:
Is this class really coming from Canonical? If not you can set (c) 2020 UBports Foundation ^^ or your name

Reply:
yes true, lets go for Ubports foundation :)